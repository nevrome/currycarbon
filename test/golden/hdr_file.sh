currycarbon "3000,30" --hdrFile actual_data/hdr_file.tsv