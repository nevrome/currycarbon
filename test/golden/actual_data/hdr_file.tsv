id	hdrSigmaLevel	hdrStartYearBCAD	hdrStopYearBCAD
1	1	-1364	-1360
1	1	-1282	-1197
1	1	-1169	-1163
1	1	-1140	-1131
1	2	-1379	-1344
1	2	-1304	-1124